src/main/java/com/cpr/codingparkrangers/configuration/SecurityConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package com.cpr.codingparkrangers.configuration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+//
+//  @Autowired
+//  private BCryptPasswordEncoder bCryptPasswordEncoder;
+//
+
```

Comment:
This is the classic SecurityConfiguration File we have been using for our previous projects. I have commented out the Encryption library instance because it will not be used until we implement users to the DB